python_modules/dagster/dagster_tests/asset_defs_tests/test_assets_job.py

```diff
@@ -893,7 +901,15 @@ def my_graph():
         graph_def=my_graph, partitions_def=StaticPartitionsDefinition(["a", "b", "c"])
     )
 
-    AssetGroup([assets_def]).build_job("abc").execute_in_process(partition_key="a")
+    assert (
```

Comment:
I think it's better to use `materialize_to_memory` here and in similar situations